.vscode/settings.json

```diff
@@ -10,15 +10,26 @@
     "AGRC",
     "GUNNISON",
     "UICLSID",
+    "atch",
```

Comment:
😆 what is an atch?

Reply:
Check [the readme](https://github.com/agrc/masquerade/blob/d6c9d716252ea6251f38fcff6560ea24a78a8139/README.md#L24). Same with `ython`...